src/tests/activity.test.js

```diff
@@ -1 +1,149 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import activity from '../models/activity';
+import activitySeed from '../seeds/activity';
+
+const mockActivity = {
+  name: 'Boxing',
+  description: 'vestibulum aliquet ultrices erat tortor sollicitudin mi non mi integer ac neque duis',
+  isActive: false,
+};
+
+const mockActivityExistingName = {
+  name: 'Chandal',
+  description: 'Any description',
+  isActive: false,
+};
+
+const mockLessName = {
+  name: '',
+  description: 'Some text',
+  isActive: true,
+};
+
+const mockLessDescription = {
+  name: 'Name',
+  description: '',
+  isActive: true,
+};
+
+const mockLessIsActive = {
+  name: 'Names',
+  description: 'some description',
+  isActive: '',
+};
+
+let firstActivityId;
+
+beforeEach(async () => {
+  await activity.collection.insertMany(activitySeed);
+  const foundFirstId = await activity.find();
+  if (foundFirstId.length > 0) {
+    // eslint-disable-next-line no-underscore-dangle
+    firstActivityId = foundFirstId[0]._id;
+  }
+});
+
+afterEach(async () => {
+  await activity.collection.deleteMany();
+});
+
+describe('GET /api/activities', () => {
+  test('The database must return 200 if is not empty of activities', async () => {
+    const response = await request(app).get('/api/activities').send();
+    expect(response.body.data.length).toBeGreaterThan(0);
+    expect(response.status).toBe(200);
+    expect(response.body.error).toBeFalsy();
+    expect(response.body).toBeDefined();
+    expect(response.body.message).toMatch('Complete activities list');
+  });
+  test('The status must be 404, if the url is wrong', async () => {
+    const response = await request(app).get('/activities').send();
+    expect(response.status).toBe(404);
+    expect(response.body.error).toBeUndefined();
+  });
+  describe('GET /api/activities', () => {
```

Comment:
Just out of curiosity, why did you add a new "describe" if it's the same as the previous one?

Reply:
I forgot to delete it, before it was at the end of the file so i needeed to describe again but i forgot to delete it when i corrected with the before each